alvr/adb/src/lib.rs

```diff
@@ -246,9 +246,9 @@ fn get_local_adb_path() -> Option<String> {
 }
 
 fn get_installation_path() -> Result<PathBuf> {
-    let mut path = std::env::current_exe()?;
-    path.pop();
-    Ok(path)
+    let root = alvr_server_io::get_driver_dir_from_registered()?;
```

Comment:
Just a warning, get_driver_dir_from_registered() is specific to SteamVR. we are moving towards adding Monado support. So you're better off taking a Filesystem as argument or just the directory path you need. at which point this function becomes redundant

Reply:
Whops it's called Layout, not Filesystem. Maybe best to rename it to FilesystemLayout, but not in this PR.